common/scala/src/main/scala/whisk/common/Logging.scala

```diff
@@ -171,6 +226,10 @@ case class LogMarkerToken(component: String, action: String, state: String) {
 
     def asFinish = copy(state = LoggingMarkers.finish)
     def asError = copy(state = LoggingMarkers.error)
+
+    def getServiceName(): String = {
+        component + "_" + action
```

Comment:
this seems like it will be high cardinality. The service name is low cardinality, and applies to all operations. action name is usually a span name as it is an operation.